tests/js/spec/views/performance/content.spec.jsx

```diff
@@ -262,10 +262,12 @@ describe('Performance > Content', function () {
     const data = initializeData(projects, {});
 
     const wrapper = mountWithTheme(
-      <PerformanceContent
-        organization={data.organization}
-        location={data.router.location}
-      />,
+      <OrganizationContext.Provider value={data.organization}>
```

Comment:
I extracted the wrapper to a function, but I didn't want to invest too much effort in trying to improve the test, because I believe it will soon be converted to RTL 😉